MANIFEST.in

```diff
@@ -2,6 +2,7 @@ include README.md
 include LICENCE
 include tox.ini
 include requirements.txt
+include requirements-dev.txt
```

Comment:
Usually you want to be able to launch the tests from that package though, so you need those requirements.